plugins/connectrpc/dart/source.yaml

```diff
@@ -0,0 +1,3 @@
+source:
+  dart_flutter:
+    name: connectrpc
```

Comment:
Double-checking, I presume we will be publishing to pub.dev?

Reply:
In other words, is this the correct place to look for new releases instead of just github?

Reply:
Updated in https://github.com/bufbuild/plugins/pull/1638/commits/a166d0d9ba4ca52046b773e53d95a6e57e933fc0